src/test/scala/no/ndla/imageapi/service/TagsServiceTest.scala

```diff
@@ -26,48 +26,4 @@ class TagsServiceTest extends UnitSuite with TestEnvironment {
     service.getISO639("http://psi.some.url.org/#ZZZ") should equal (None)
```

Comment:
Gjorde flere forsøk på det, men gikk meg bort i `Try`-håndteringen i de to funksjonene. Men skal ta en kikk på linken og gjøre et forsøk til :)